forc-plugins/forc-client/src/util/pkg.rs

```diff
@@ -1,8 +1,145 @@
 use anyhow::Result;
 use forc_pkg::manifest::GenericManifestFile;
 use forc_pkg::{self as pkg, manifest::ManifestFile, BuildOpts, BuildPlan};
-use pkg::{build_with_options, BuiltPackage};
+use forc_util::user_forc_directory;
+use pkg::{build_with_options, BuiltPackage, PackageManifestFile};
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::PathBuf;
 use std::{collections::HashMap, path::Path, sync::Arc};
+use sway_utils::{MAIN_ENTRY, MANIFEST_FILE_NAME, SRC_DIR};
+
+/// The name of the folder that forc generated proxy contract project will reside at.
+pub const PROXY_CONTRACT_FOLDER_NAME: &str = ".generated_proxy_contracts";
+/// Forc.toml for the default proxy contract that 'generate_proxy_contract_src()' returns.
+pub const PROXY_CONTRACT_FORC_TOML: &str = r#"
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "proxy_contract"
+
+[dependencies]
+standards = { git = "https://github.com/FuelLabs/sway-standards/", tag = "v0.5.0" }
+"#;
+
+/// Generates source code for proxy contract owner set to the given 'addr'.
+pub(crate) fn generate_proxy_contract_src(addr: &str, impl_contract_id: &str) -> String {
+    format!(
+        r#"
+contract;
+
+use std::execution::run_external;
+use standards::src5::{{AccessError, SRC5, State}};
+use standards::src14::SRC14;
+
+/// The owner of this contract at deployment.
+const INITIAL_OWNER: Identity = Identity::Address(Address::from({addr}));
+
+// use sha256("storage_SRC14") as base to avoid collisions
+#[namespace(SRC14)]
```

Comment:
This syntax is deprecated. There's a new syntax for namespaces: https://github.com/FuelLabs/sway/pull/6064